addons/mail/static/tests/core/common/store_service.test.js

```diff
@@ -73,12 +73,9 @@ test("store.insert different PY model having same JS model", async () => {
         ],
     };
 
-    const { Thread: threads } = store.insert(data);
```

Comment:
Note: not directly related to the PR, but as the goal is to remove store insert result, better not use in the tests.